index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="initial-scale=1, width=device-width" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" />
+  </head>
+  <body>
+    <header class="header">
+      <a href="https://www.linkedin.com/in/qaisar-abbas-21a93840/" class="logo"><img  src="./logo.png" alt="logo" class="logo-image"></a>
+      <nav class="navbar">
+         <div class="bar" href="#"></div>
+         <div class="bar"href="#"></div>
+         <div class="bar"href="#"></div>
+      </nav>
+    </header>
+    <section class="main container">
+      <div class="headline">
+        <b class="primary-text">
+          <span>
+            <h2 class="im-qaisar">I’m Qaisar.</h2>
+            <p class="im-qaisar">Glad to see you!</p>
```

Comment:
- [ ] As requested by the previous reviewer, I recommend using the <h1> tag for your main project title. The <h1>and replace the <p> tag with <h2>tag. this is typically used for the main heading of a webpage and should only be used once for each page to maintain proper semantic structure. By using the <h1> tag, you can enhance the semantic meaning of your project and improve its overall structure.